File: src/nl/stenwessel/texifyidea/psi/LatexElementType.java
package nl.stenwessel.texifyidea.psi;

import com.intellij.psi.tree.IElementType;
import nl.stenwessel.texifyidea.LatexLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

/**
 * @author Sten Wessel
 */
public class LatexElementType extends IElementType {

    public LatexElementType(@NotNull @NonNls String debugName) {
        super(debugName, LatexLanguage.INSTANCE);
    }
}

File: src/nl/hannahsten/texifyidea/psi/LatexElementType.kt
package nl.hannahsten.texifyidea.psi

import com.intellij.psi.tree.IElementType
import nl.hannahsten.texifyidea.grammar.LatexLanguage
import org.jetbrains.annotations.NonNls

/**
 * @author Sten Wessel
 */
class LatexElementType(@NonNls debugName: String) : IElementType(debugName, LatexLanguage)